src/util/workers.c

```diff
@@ -30,33 +31,58 @@ static void yieldCallback(void *yieldCtx) {
   RedisModule_Yield(ctx, REDISMODULE_YIELD_FLAG_CLIENTS, NULL);
 }
 
+/* Configure here anything that needs to know it can use the thread pool */
+static void workersThreadPool_OnTurnOn(size_t new_num) {
+  // Log that we've enabled the thread pool.
+  RedisModule_Log(RSDummyContext, "notice", "Enabled workers threadpool of size %lu", new_num);
+  // Change VecSim write mode temporarily for fast RDB loading of vector index (if needed).
+  VecSim_SetWriteMode(VecSim_WriteAsync);
+}
+
+/* Configure here anything that needs to know it cannot use the thread pool anymore */
+static void workersThreadPool_OnTurnOff(size_t old_num) {
+  RedisModule_Log(RSDummyContext, "notice", "Disabled workers threadpool of size %lu", old_num);
```

Comment:
create/add/remove also logs number of threads
make sure we don't have redundant logs here

Reply:
Will verify. These logs already removed some duplications with other places

Reply:
After verification, this log is on a `notice` level and informs about stop/start using the thread pool. The logs in `thpool.c` are on a `verbose` level and log any change in the number of threads. I think both are valid